bagpipes/fitting/check_priors.py

```
@@ -136,7 +136,7 @@ def _update_model_components(self, param):
             elif len(split) == 2:
                 if "dirichlet" in split[1]:
                     if split[0] not in dirichlet_comps:
-                        dirichlet_comps.append(split[0])
+                        dirichlet_compxs.append(split[0])
```

Ruff (0.15.20)

[error] Undefined name `dirichlet_compxs`

(F821)